core/src/database/migrations.cpp

```diff
@@ -586,7 +586,7 @@ namespace ledger {
                     "uid VARCHAR(255) NOT NULL PRIMARY KEY REFERENCES accounts(uid) ON DELETE CASCADE ON UPDATE CASCADE,"
                     "wallet_uid VARCHAR(255) NOT NULL REFERENCES wallets(uid) ON DELETE CASCADE ON UPDATE CASCADE,"
                     "idx INTEGER NOT NULL,"
-                    "address VARCHAR(255) NOT NULL"
+                    "public_key VARCHAR(255) NOT NULL"
```

Comment:
Wait, is it safe to modify an “old” transaction like that?

Reply:
I'm interesting too. Should we add a new migration with that column renaming ?

Reply:
I did it in purpose because nobody is using Tezos yet :trollface: so instead of adding a migration for that ...